src/components/NewsFinder/NewsFinder.css

```diff
@@ -16,9 +16,9 @@
 }
 @media (max-width: 590px) {
     .finder__container{
-        margin: 0 16px;
+        margin: 0;
```

Comment:
**Might be better**

You can add an `Escape` listener like this not to repeat it in every opening in `App`:

```jsx
 useEffect(() => {
      const closeByEscape = (e) => {
        if (e.key === 'Escape') {
          closeAllPopups();
        }
      }

      document.addEventListener('keydown', closeByEscape)
      
      return () => document.removeEventListener('keydown', closeByEscape)
  }, [])

```

Here you define function `closeByEscape` inside the `useEffect` so that it could maintain a fixed reference if the component rerenders.
And of course we need to return a `clean up` function to `remove` the listener if the component unmounts.
And don’t forget about an `empty dependencies array` in order to add the listener only once.